libbeat/otelbeat/providers/common_provider.go

```diff
@@ -0,0 +1,68 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Wdyt of renaming this file to `providers.go`?

Reply:
+1